Makefile

```diff
@@ -184,13 +184,9 @@ sassc: $(SASSC_EXE)
 
 $(SASSC_EXE): libsass build-$(BUILD)
 
-$(DESTDIR)$(PREFIX)/:
-	$(MKDIR) $(DESTDIR)$(PREFIX)
-
-$(DESTDIR)$(PREFIX)/bin/:
-	$(MKDIR) $(DESTDIR)$(PREFIX)/bin
-
 $(DESTDIR)$(PREFIX)/bin/%: bin/%
+	-$(MKDIR) $(DESTDIR)$(PREFIX)
+	-$(MKDIR) $(DESTDIR)$(PREFIX)/bin
```

Comment:
This change now fails under MSys2 mingw if directory already exists:
mkdir: cannot create directory ‘/usr/local/bin’: File exists
Makefile:188: recipe for target '/usr/local/bin/sassc.exe' failed